package.json

```diff
@@ -31,19 +31,18 @@
     "analytics"
   ],
   "dependencies": {
+    "@adobe/node-fetch-retry": "^1.1.1",
     "@expo/bunyan": "^4.0.0",
     "@segment/loosely-validate-event": "^2.0.0",
-    "axios": "^0.21.1",
-    "axios-retry": "^3.0.2",
     "lodash.isstring": "^4.0.1",
     "md5": "^2.2.1",
     "ms": "^2.0.0",
     "remove-trailing-slash": "^0.1.0",
-    "serialize-javascript": "^5.0.1",
```

Comment:
was apparently only used in the persistence queue code which has already been removed. just using normal JSON.stringify in the HTTP request